includes/sys_template.php

```diff
@@ -489,4 +489,10 @@ function ReplaceSmilies($neueckig) {
   
   return $neueckig;
 }
+/**
+ * Rendert Google reCaptcha
+ */
+function reCaptcha() {
+  return '<div class="g-recaptcha" data-sitekey="6LeGiyITAAAAAGG2-A9yM47fnB0IRwET_cOunvgf"></div>';
```

Comment:
Use key from `config.php`! #142 
You may move the function to `sys_recaptcha.php`, but `sys_template.php` is ok too.